tests/ignite/handlers/test_state_param_scheduler.py

```diff
@@ -95,7 +96,11 @@ def test_pwlinear_scheduler_step_constant(max_epochs, milestones_values):
     )
     linear_state_parameter_scheduler.attach(engine, Events.EPOCH_COMPLETED)
     engine.run([0] * 8, max_epochs=max_epochs)
-    torch.testing.assert_allclose(getattr(engine.state, "pwlinear_scheduled_param"), milestones_values[0][1])
+    if Version(torch.__version__) <= Version("1.7.0"):
+        torch_testing_assert_close = torch.testing.assert_allclose
+    else:
+        torch_testing_assert_close = torch.testing.assert_close
+    torch_testing_assert_close(getattr(engine.state, "pwlinear_scheduled_param"), float(milestones_values[0][1]))
```

Comment:
As I suggested previously, the point is to define `torch_testing_assert_close` **once** in the global scope